docs/src/tutorials/basic/material_library.rst

```diff
@@ -1,4 +1,4 @@
-Materials
+Defining materials
 =========
 
 In order to support a wide range of applications (and to build complex `assemblies`_) there are a few different types of material that can be utilised in :py:mod:`easyreflectometry`. 
```

Comment:
The next two lines have some spelling mistakes ("enabel", "mdoule"), but I can't seem to suggest a change here since they're not part of the PR.